### PR TITLE
[SYCL] Prepare FilterSelector tests for migrating to sycl::oneapi

### PR DESCRIPTION
**Pull request by dm-vodopyanov**
This patch prepares FilterSelector tests for transitioning from
`sycl::ONEAPI` to `sycl::oneapi` namespace.

### CODE REVIEW

**Comment by dm-vodopyanov** [comment]
This PR should be merged before the PR for impl https://github.com/intel/llvm/pull/4014.

File: SYCL/FilterSelector/reuse.cpp [hunk 1]
```diff
@@ -15,6 +15,8 @@
 #include <CL/sycl.hpp>
 
 using namespace cl::sycl;
+// change to 'using namespace cl::sycl::oneapi' after PR intel/llvm:4014 is
```
Comment (s-kanaev) [inline SYCL/FilterSelector/reuse.cpp]: Is it a TODO or description of what's done?

File: SYCL/FilterSelector/select.cpp [hunk 2]
```diff
@@ -67,60 +69,62 @@ int main() {
 
   if (!CPUs.empty()) {
     std::cout << "Test 'cpu'";
-    device d1(filter_selector("cpu"));
+    // change all occurrences of filter_selector to 'filter_selector' or
+    // 'oneapi::filter_selector' after PR intel/llvm:4014 is merged
```
Comment (s-kanaev) [inline SYCL/FilterSelector/select.cpp]: Same question here.

File: SYCL/FilterSelector/reuse.cpp (hunk 1 above)
Comment (dm-vodopyanov) [reply SYCL/FilterSelector/reuse.cpp]: Sorry, missed `TODO: `, fixed

**Review by vladimirlaz — APPROVED** [review]

**Pull request merged**